docker-compose.yaml

```diff
@@ -55,6 +55,34 @@ services:
       deploy-sequencer-contracts:
         condition: service_completed_successfully
 
+  upgrade-prover-contracts:
+    image: ghcr.io/espressosystems/espresso-sequencer/deploy:${DOCKER_TAG:-main}
+    command: deploy --use-mock --upgrade-light-client-v2 --use-multisig --dry-run
```

Comment:
Is this service added to see if we can run `deploy` with those argument in the docker container successfully?

Reply:
I'm thinking maybe we can just put an assert into the docker build? I think this currently mainly tests that we can run the typescript code and the argument parsing is correct. It's not  exactly what I had in mind with the dry-run flag but yeah it's not a bad idea to have this either.

If we're not checking the output we get from the typescript script I would probably add a subcommand to the deploy binary like `verify-installation` and call it at the end of the docker build (inside the dockerfile) to ensure that the rust script in the container can execute the typescript script.